test/init-ssr.spec.js

```diff
@@ -71,8 +71,6 @@ describe("init() method in Node/SSR environment (should not crash)", () => {
           authentication: {
             firstFactors: [],
             secondFactors: [],
-            isFirstFactorTokenPresent: false,
-            isEnabled: false,
```

Comment:
These are not part of the `/mode` response, so removing them for clarity. They do not affect the tests.